src/rt/lifetime.d

```diff
@@ -1823,15 +1819,7 @@ byte[] _d_arrayappendcTX(const TypeInfo ti, ref byte[] px, size_t n)
                 }
 
                 // couldn't do it, reallocate
-                info = GC.qalloc(newcap + LARGEPAD, info.attr);
-                __setArrayAllocLength(info, newsize, isshared);
-                if(!isshared)
-                    __insertBlkInfoCache(info, bic);
-                auto newdata = cast(byte *)info.base + LARGEPREFIX;
-                memcpy(newdata, px.ptr, length * sizeelem);
-                // do postblit processing
-                __doPostblit(newdata, length * sizeelem, ti.next);
-                (cast(void **)(&px))[1] = newdata;
```

Comment:
This is the original code that runs during the `split` operation. I verified by `printf`during the test. After the pull, it's a goto to L2. Compare with that code. It's identical except for using info to determine whether to do LARGEPAD or assuming LARGEPAD, and doing a prefix, or assuming a prefix. everything else is identical.

In the case of slist test, the case that this code was intended to fix doesn't occur (appending to a small slice of a PAGE sized+ block), so the result should also be identical. I've looked at this code about 10 times, there is no difference I can see. Something other than this pull has to be the cause.